test/integration/install/test_install.py

```diff
@@ -645,3 +646,94 @@ def test_install_tt_fail_exit_code_dependency_check(
     )
     install_process_rc = install_process.wait()
     assert install_process_rc == 1
+
+
+@pytest.mark.slow
+@pytest.mark.parametrize("old_tarantool_version, expected_install_msg, is_interactive", [
+    pytest.param("7192bf6", "Found newest commit of tarantool in master", True),
+    pytest.param("master", "tarantool_master version of tarantool already exists", False),
+    pytest.param("7192bf6", "tarantool_master version of tarantool already exists", False)
+])
+def test_install_tarantool_fetch_latest_version(
+        tt_cmd,
+        tmp_path,
+        old_tarantool_version,
+        expected_install_msg,
+        is_interactive):
+    config_path = os.path.join(tmp_path, config_name)
+    # Create test config.
+    with open(config_path, "w") as f:
+        yaml.dump({"env": {"bin_dir": "", "inc_dir": "./my_inc"}}, f)
+
+    tmp_path_without_config = tempfile.mkdtemp()
+
+    # Install not latest version of tarantool by commit hash.
+    install_cmd = [tt_cmd, "--cfg", config_path, "install",
+                   "-f", "tarantool", old_tarantool_version, "--dynamic"]
+    instance_process_rc, _ = run_command_and_get_output(install_cmd,
+                                                        cwd=tmp_path_without_config,
+                                                        stdout=subprocess.DEVNULL)
+    assert instance_process_rc == 0
+
+    # Renaming installed 'old' tarantool to 'master'
+    # and change symlink to old version to simulate
+    # that this is the latest version.
+    os.rename(os.path.join(tmp_path, "bin", "tarantool_" +
+              old_tarantool_version), os.path.join(tmp_path, "bin", "tarantool_master"))
+    os.remove(os.path.join(tmp_path, "bin", "tarantool"))
+    os.symlink(os.path.join(tmp_path, "bin", "tarantool_master"),
+               os.path.join(tmp_path, "bin", "tarantool"))
+    os.remove(os.path.join(tmp_path, "my_inc", "include", "tarantool"))
+    os.rename(os.path.join(tmp_path, "my_inc", "include",
+              "tarantool_" + old_tarantool_version),
+              os.path.join(tmp_path, "my_inc", "include", "tarantool_master"))
+
+    # Installing newest version.
+    install_cmd = [tt_cmd, "--cfg", config_path]
+    if not is_interactive:
+        install_cmd.append("--no-prompt")
+    install_cmd.extend(["install", "-f", "tarantool", "master", "--dynamic"])
```

Comment:
I understand the importance of integration testing. But each test that performs tarantool installation increases the whole test time significantly. As I remember, we don't test `--reinstall` or `--force` options for example. I believe that it is enough to leave at least simple `tt install tarantool` command test and check other `install tarantool` options manually. For, sure the tests for commit hash checking, declining update should still be here.
@oleg-jukovec what do you think?

Reply:
Don't we have any label for long tests?

Reply:
Yes, we have - `slow`. full ci takes ~ 1 hour and each `install` test adds 5-10 minutes to this time. From my perspective, it's simply not worth it.